### PR TITLE
#809 - Documentation refresh for developer instructions

### PR DESCRIPTION
**Pull request by gisjedi**

### CODE REVIEW

File: scale/README.md [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+
+- Running Docker Community Edition 1.10+ Engine (use Docker for Windows or Mac on those platforms)
+- Python 2.7.x
+
+The core Scale team uses JetBrains PyCharm or Cloud9 IDE for development. These are in no way required but are
+our preferred choices.
+
+#### Cloud IDEs
+
+Cloud9 and Codeanywhere comes with built in support for Postgres / PostGIS databases, making development of Scale both
```
Comment (mikenholt) [inline scale/README.md]: Suggest links to each service 
https://c9.io/
https://codeanywhere.com/

File: scale/environment/win-init.bat [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+echo "        'PASSWORD': 'scale'," >> scale/local_settings.py
+echo "        'HOST': 'localhost'," >> scale/local_settings.py
+echo "        'PORT': '${SCALE_DB_PORT}'," >> scale/local_settings.py
+echo "        'TEST': {'NAME': 'test_scale'}," >> scale/local_settings.py
+echo "    }," >> scale/local_settings.py
+echo "}" >> scale/local_settings.py
+
+REM Initialize virtual environment
+virtualenv environment\scale
+environment\scale\bin\pip install -r pip\requirements.txt
```
Comment (dfaller) [inline scale/environment/win-init.bat]: `bin` needs changed to `Scripts`

File: scale/environment/win-init.bat [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+REM Load up database with schema migrations to date and fixtures
+environment\scale\bin\python manage.py migrate
+environment\scale\bin\python manage.py load_all_data
```
Comment (dfaller) [inline scale/environment/win-init.bat]: `bin` needs changed to `Scripts`

File: scale/environment/win-init.bat [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+echo "    'default': {" >> scale/local_settings.py
+echo "        'ENGINE': 'django.contrib.gis.db.backends.postgis'," >> scale/local_settings.py
+echo "        'NAME': 'scale'," >> scale/local_settings.py
+echo "        'USER': 'scale'," >> scale/local_settings.py
+echo "        'PASSWORD': 'scale'," >> scale/local_settings.py
+echo "        'HOST': 'localhost'," >> scale/local_settings.py
+echo "        'PORT': '${SCALE_DB_PORT}'," >> scale/local_settings.py
+echo "        'TEST': {'NAME': 'test_scale'}," >> scale/local_settings.py
+echo "    }," >> scale/local_settings.py
+echo "}" >> scale/local_settings.py
```
Comment (dfaller) [inline scale/environment/win-init.bat]: This does not seem to work well for generating the local settings file. Here is what it creates:

```
```

**Pull request merged**